Assignment3/protocol.py

```diff
@@ -170,13 +152,8 @@ def SetSharedKey(self, secret):
     # RETURNS A PYTHON utf-8 string
     def EncryptAndProtectMessage(self, plain_text):
         if self.ping_pong_done and self._session_key != None:
-            print("Secure session key exists.. Encrypting with session key.")
             return self._encrypt(plain_text, self._session_key)
-        elif self._init_shared_key != None:
```

Comment:
If this is gone then I think the initial auth messages will not be encrypted. Did you consider this? Am I missing something?

Reply:
this would be same for L206